packages/fxa-settings/src/components/HeaderContent/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+export type HeaderContentProps = {
+  headingWithDefaultServiceFtlId: string;
+  headingWithCustomServiceFtlId: string;
+  headingText: string;
+  serviceName?: string;
+};
+
+const HeaderContent = ({
+  headingWithDefaultServiceFtlId,
+  headingWithCustomServiceFtlId,
+  headingText,
+  serviceName,
+}: HeaderContentProps) => {
+  return (
+    <header className="card-header">
+      {serviceName ? (
+        <Localized id={headingWithCustomServiceFtlId}>
+          <h1 className="card-heading">
+            {headingText}
+            <span className="card-subheading">{`\nto continue to ${serviceName}`}</span>
+          </h1>
+        </Localized>
+      ) : (
+        <Localized id={headingWithDefaultServiceFtlId}>
+          <h1 className="card-heading">
+            {headingText}
+            <span className="card-subheading">
+              {`\nto continue to account settings`}
```

Comment:
Instead of `\n`, we should set this to `display: block`. It looks like you can just use the `card-subheader` class though, which already has that applied.

Reply:
I initially just had `display:block`, but was seeing some weirdness in the test with the strings getting smushed together. Maybe it doesn't matter though, and I can just set the test to expect "Reset passwordto continue to account settings"?

Reply:
Ah, I think you might need to use `{' '}`?